### PR TITLE
Update data.yml

### PR DESCRIPTION
**Pull request by Haselnussbomber**
Tried to name some of the most xref'd functions, but didn't get many of them and got sidetracked a lot.
Hope I got things right, it's 6 am. 😪

### CODE REVIEW

File: FFXIVClientStructs/FFXIV/Component/GUI/AtkComponentButton.cs [hunk 1]
```diff
@@ -31,6 +31,9 @@ public bool IsChecked {
     [MemberFunction("E8 ?? ?? ?? ?? 48 8D 05 ?? ?? ?? ?? 88 9F ?? ?? ?? ??")]
     public partial void Ctor();
 
+    [MemberFunction("48 89 5C 24 ?? 57 48 83 EC 20 48 8B D9 48 8B FA 48 8B 89 ?? ?? ?? ?? 48 85 C9 0F 84 ?? ?? ?? ?? 48 85 D2 74 ?? 0F B6 81"), GenerateStringOverloads]
+    public partial void SetText(byte* text);
```
Comment (wolfcomp) [inline FFXIVClientStructs/FFXIV/Component/GUI/AtkComponentButton.cs:35]: This function sets the text pointer to what is sent in here and can cause memory corruption with `GenerateStringOverloads` as it calls `AtkTextNode.SetText` inside this function

Ref: #1040

**Review by wolfcomp — CHANGES_REQUESTED** [review]
Need to change this to not cause another headache from past mistakes.

File: FFXIVClientStructs/FFXIV/Component/GUI/AtkComponentButton.cs (hunk 1 above)
Comment (Haselnussbomber) [reply FFXIVClientStructs/FFXIV/Component/GUI/AtkComponentButton.cs:35]: Thanks. Removed the attribute.

File: FFXIVClientStructs/FFXIV/Component/GUI/AtkComponentButton.cs [hunk 2]
```diff
@@ -31,6 +31,9 @@ public bool IsChecked {
     [MemberFunction("E8 ?? ?? ?? ?? 48 8D 05 ?? ?? ?? ?? 88 9F ?? ?? ?? ??")]
     public partial void Ctor();
 
+    [MemberFunction("48 89 5C 24 ?? 57 48 83 EC 20 48 8B D9 48 8B FA 48 8B 89 ?? ?? ?? ?? 48 85 C9 0F 84 ?? ?? ?? ?? 48 85 D2 74 ?? 0F B6 81")]
```
Comment (wolfcomp) [inline FFXIVClientStructs/FFXIV/Component/GUI/AtkComponentButton.cs:34]:
```suggestion
        /// <summary>
    /// Set the text of the component button node.
    /// The game assumes the pointer passed to this function will stay alive. See <see href="https://github.com/aers/FFXIVClientStructs/issues/1040">here</see> for more information.
    /// </summary>
    /// <param name="str">Null-terminated UTF-8 string buffer to set the text to.</param>
    [MemberFunction("48 89 5C 24 ?? 57 48 83 EC 20 48 8B D9 48 8B FA 48 8B 89 ?? ?? ?? ?? 48 85 C9 0F 84 ?? ?? ?? ?? 48 85 D2 74 ?? 0F B6 81")]
```

**Review by wolfcomp — APPROVED** [review]

**Pull request merged**